pkgs/development/python-modules/hypercorn/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy27, isPy35, isPy36
+, pythonOlder
+, typing-extensions
+, wsproto
+, toml
+, h2
+, priority
+}:
+
+buildPythonPackage rec {
+  pname = "Hypercorn";
+  version = "0.11.2";
+  disabled = isPy27 || isPy35 || isPy36;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "5ba1e719c521080abd698ff5781a2331e34ef50fc1c89a50960538115a896a9a";
+  };
+
+  propagatedBuildInputs = [ wsproto toml h2 priority ]
+  ++ lib.optionals (pythonOlder "3.8") [ typing-extensions ];
+
+  doCheck = false;
```

Comment:
If there are no tests included, please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.